assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -246,4 +249,8 @@ export const moveItemDown = (itemsById: ItemsById, id: string): ItemsById => {
     order += 1;
   });
   return newItemsById;
-};
\ No newline at end of file
+};
+
+export function isHarvestable(params: RouterParams) {
```

Comment:
I prefer isHarvestablePhase instead of isHarvestable

Reply:
We discussed it with Vincent. We chose `isHarvestable` because in the future, we will have to change this function to use the info on the idea to know if it is an idea from thread/multicolumns (we will have phases with ideas of different types (survey/thread/votes))

Reply:
Indeed, we might not talk about phases anymore in the long term